content-src/components/LinkMenuButton/LinkMenuButton.scss

```diff
@@ -6,7 +6,7 @@
   width: $link-menu-button-size;
   height: $link-menu-button-size;
   background-color: $white;
-  background-image: url('img/glyph-more-16.svg');
+  background-image: url($image-path + 'glyph-more-16.svg');
```

Comment:
`'#{$foo}bar'` does look a little nicer, but it's fine if we leave this as a follow-up